### PR TITLE
Merge README backend changes

### PR DESCRIPTION
**Pull request by chenriksson**
These are lauren's last changes where we removed storage of the converted readme.html files.

This needs to be merged for the Gallery README support to work, otherwise backend with fail with 404 storage exception.

### CODE REVIEW

File: src/HandlePackageEdits/HandlePackageEdits.Job.cs [hunk 1]
```diff
@@ -282,13 +273,6 @@ public override void Init(IDictionary<string, string> jobArgsDictionary)
                     await readMeMDBlob.activeSnapshot.DeleteAsync();
                     Logger.LogInformation("Deleted snapshot ReadMe Md {ActiveReadMeMdUri} for {EditId} {EditVersion}.", readMeMDBlob.activeSnapshot.Uri.AbsoluteUri, edit.Id, edit.Version);
                 }
-
-                if (readMeHTMLBlob.activeSnapshot != null)
-                {
-                    Logger.LogInformation("Deleting snapshot ReadMe Html {ActiveReadMeHtmlUri} for {EditId} {EditVersion}.", readMeHTMLBlob.activeSnapshot.Uri.AbsoluteUri, edit.Id, edit.Version);
-                    await readMeHTMLBlob.activeSnapshot.DeleteAsync();
-                    Logger.LogInformation("Deleted snapshot ReadMe Html {ActiveReadMeHtmlUri} for {EditId} {EditVersion}.", readMeHTMLBlob.activeSnapshot.Uri.AbsoluteUri, edit.Id, edit.Version);
-                }
```
Comment (cristinamanu) [inline src/HandlePackageEdits/HandlePackageEdits.Job.cs]: Why do we delete this sequence? What is the change in behavior?

File: src/HandlePackageEdits/HandlePackageEdits.Job.cs [hunk 2]
```diff
@@ -265,11 +260,7 @@ public override void Init(IDictionary<string, string> jobArgsDictionary)
                     // Delete pending ReadMes
                     Logger.LogInformation("Deleting pending ReadMe for {EditId} {EditVersion} from {PendingReadMeMdUri}", edit.Id, edit.Version, readMeMDBlob.pendingBlob.Uri.AbsoluteUri);
                     await readMeMDBlob.pendingBlob.DeleteIfExistsAsync();
-                    Logger.LogInformation("Deleted pending ReadMe for {EditId} {EditVersion} from {PendingReadMeMdUri}", edit.Id, edit.Version, readMeMDBlob.pendingBlob.Uri.AbsoluteUri);
-
-                    Logger.LogInformation("Deleting pending ReadMe for {EditId} {EditVersion} from {PendingReadMeHtmlUri}", edit.Id, edit.Version, readMeHTMLBlob.pendingBlob.Uri.AbsoluteUri);
-                    await readMeHTMLBlob.pendingBlob.DeleteIfExistsAsync();
-                    Logger.LogInformation("Deleted pending ReadMe for {EditId} {EditVersion} from {PendingReadMeHtmlUri}", edit.Id, edit.Version, readMeHTMLBlob.pendingBlob.Uri.AbsoluteUri);
```
Comment (cristinamanu) [inline src/HandlePackageEdits/HandlePackageEdits.Job.cs]: Why the pending blob is not deleted anymore?

Comment (cristinamanu) [reply src/HandlePackageEdits/HandlePackageEdits.Job.cs]: Oh - I think we had it twice ...

File: src/HandlePackageEdits/HandlePackageEdits.Job.cs [hunk 3]
```diff
@@ -36,8 +36,7 @@ internal class Job : JobBase
 
         private const string ReadMeChanged = "changed";
         private const string ReadMeDeleted = "deleted";
-        private const string MarkdownExtension = "md";
-        private const string HtmlExtension = "html";
+        private const string MarkdownExtension = ".md";
```
Comment (cristinamanu) [inline src/HandlePackageEdits/HandlePackageEdits.Job.cs]: Was this change tested? Is there a deployment ordering necessary between this job and gallery?

**Comment by chenriksson** [comment]
@cristinamanu Originally we would convert the readme.md to html at upload, and save both to storage for better performance. We changed to only store md and convert to html at display time, in case there is a security vulnerability that we need to patch with the conversion.

This change removes references to the readme html that we no longer store.

**Comment by joelverhagen** [comment]
It's a bit spooky that there are no test changes due to this change. Are there unit tests at all?

**Comment by chenriksson** [comment]
@joelverhagen AFAIK, there's no unit tests for this job at all

File: src/HandlePackageEdits/HandlePackageEdits.Job.cs (hunk 3 above)
Comment (chenriksson) [reply src/HandlePackageEdits/HandlePackageEdits.Job.cs]: This is the branch the explorers used for their demo, and that I've tested with locally.  Realized this was missing when validating on dev environment.

**Pull request merged**